uwrt_mars_rover/CMakeLists.txt

```diff
@@ -1,4 +1,35 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(uwrt_mars_rover)
-find_package(catkin REQUIRED)
-catkin_metapackage()
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+# uncomment the following section in order to fill in
+# further dependencies manually.
+# find_package(<dependency> REQUIRED)
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  # the following line skips the linter which checks for copyrights
+  # uncomment the line when a copyright and license is not present in all source files
+  #set(ament_cmake_copyright_FOUND TRUE)
+  # the following line skips cpplint (only works in a git repo)
+  # uncomment the line when this package is not in a git repo
+  #set(ament_cmake_cpplint_FOUND TRUE)
+  ament_lint_auto_find_test_dependencies()
```

Comment:
The tests that get run here are based off the test dependencies specified in the xml. The proper way to disable running thise tests is just not to include the `ament_lint_common` test depend and include each linter individually there

Reply:
https://github.com/ament/ament_lint/blob/master/ament_lint_common/doc/index.rst

Reply:
> https://github.com/ament/ament_lint/blob/master/ament_lint_common/doc/index.rst



> The tests that get run here are based off the test dependencies specified in the xml. The proper way to disable running thise tests is just not to include the `ament_lint_common` test depend and include each linter individually there

Oh nice I will do that then